javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/ignore/GlobalIgnoredTypesConfigurer.java

```diff
@@ -156,6 +156,7 @@ private static void configureIgnoredTasks(IgnoredTypesBuilder builder) {
     // TODO Workaround for
     // https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/787
     builder.ignoreTaskClass("org.apache.tomcat.util.net.NioEndpoint$SocketProcessor");
+    builder.ignoreTaskClass("org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor");
```

Comment:
is this intended to be part of this PR?

Reply:
it probably depended on the order of the tests which probably depends on the order of the methods returned from reflection which can be somewhat unpredictable and depend on how many method you have in the class etc.